patronload/email.py

```diff
@@ -44,19 +39,8 @@ def populate(  # noqa pylint R0913 too many arguments
         self["From"] = from_address
         self["To"] = to_addresses
         self["Subject"] = subject
-        if cc:
-            self["Cc"] = cc
-        if bcc:
-            self["Bcc"] = bcc
-        if reply_to:
-            self["Reply-To"] = reply_to
         if body:
             self.set_content(body)
-        if attachments:
-            for attachment in attachments:
-                self.add_attachment(
-                    attachment["content"], filename=attachment["filename"]
-                )
```

Comment:
Was it determined that we don't need cc, bcc, reply_to, and attachments?

Reply:
This is just for sending the logs and none of those fields are used. I believe I stole this from another repo without any modifications and I don't see a use case where this app would start using those fields